src/main/java/net/fabricmc/tinyremapper/api/ClassHeader.java

```diff
@@ -0,0 +1,30 @@
+package net.fabricmc.tinyremapper.api;
+
+import java.util.List;
+
+public interface ClassHeader {
+	Classpath getClasspath();
+
+	int getAccess();
+
+	String getName();
+
+	String getSuperName();
+
+	String getSignature();
+
+	List<String> getInterfaceList();
+
+	/**
+	 * get field by name and descriptor, this is not cached.
+	 * @param desc can be null
+	 */
+	MemberHeader getField(String name, String desc);
+
+	/**
+	 * get method by name and descriptor, this is not cached.
+	 */
+	MemberHeader getMethod(String name, String desc);
```

Comment:
Since you use `resolve`, which uses the exact match. The documentation here is not correct anymore.